src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline

```diff
@@ -717,9 +717,6 @@ public class LibraryServiceSettings extends ServiceApiSettings {
           addCommentsSettings,
           getBookFromArchiveSettings,
           updateBookIndexSettings,
-          streamShelvesSettings,
```

Comment:
ummm before the template was referring  xsettingsClass.callSettings which contains all settings but I corrected it to xsettingsClass.unaryCallSettings which only contains unary settings. Does it answer you question?